src/main/proto/messages.proto

```diff
@@ -51,6 +51,7 @@ message StatisticSetRecord {
     string metric = 3;
     string period = 4;
     string period_start = 5;
+    string min_request_time = 9;
```

Comment:
Is the intent that this field will be an ISO-8601 date-time field? e.g. like `period_start`

I would like to ensure the name clearly indicates that this is not MAD's request time to CAGG, but rather the client's request time to MAD.

e.g. `client_minimum_request_time`

We used `request_millis_since_epoch` in the client API:

https://github.com/InscopeMetrics/client-protocol/blob/master/src/main/proto/client.v3.proto#L38

However, I don't think we should try to change the naming convention or date-time format in this protocol (or the client one) to align at this time. So Let's assume here it's ISO-8601 and less strict name. I suppose if we really want it to be the same it could be `client_minimum_request_start`.

@BrandonArp what do you think?

Reply:
> Is the intent that this field will be an ISO-8601 date-time field? e.g. like period_start

Yes.

Reply:
I'm also a fan of client_minimum_request_time. And could you please add a comment about what it is for?